src/import.ml

```diff
@@ -14,11 +14,22 @@ let property_exists json property =
 
 include Base
 
+module List = struct
+  let sort' lst ~compare =
+    List.sort lst ~compare:(fun a b -> compare a b |> Ordering.to_int)
```

Comment:
Why can't we use the original `List.sort`?

Reply:
The `List.sort` that comes with Base uses an `int`-based compare, so I'm adding a sort that uses `Ordering.t`. 

Do you want me to shadow int-based `List.sort` of `Base`? (there are other `int`-compare-based functions in Base, which either need to be also shadowed or we should live with inconsistencies)

Reply:
I see. IMO we should live with the inconsistencies and just report them to the base bug tracker.

Reply:
I meant that if I shadow a single `List.sort` with our Ordering-based sort, all other Base functions would still use int-based compares. This is an inconsistency we create ourselves by deviating from Base's standard int-based compare. I can still do the shadowing.

re: traversal comments

I got what you meant and will fix them :-)